packages/components/filters/src/filter-menu/header/header.tsx

```diff
@@ -1,5 +1,73 @@
-function Header() {
-  return <div>header</div>;
-}
+import { useState, ReactNode } from 'react';
+import IconButton from '@commercetools-uikit/icon-button';
+import Spacings from '@commercetools-uikit/spacings';
+import { SortingIcon } from '@commercetools-uikit/icons';
+import { designTokens } from '@commercetools-uikit/design-system';
+import { css } from '@emotion/react';
+
+type TOption = {
+  value: string;
+  label: string;
+};
+
+type THeaderProps = {
+  headerLabel: ReactNode;
+  operandOptions?: Array<TOption>;
+  renderOperandsInput?: boolean;
+  /**
+   * the function to render the operator input
+   */
+  renderOperatorsInput?: () => ReactNode;
+  /**
+   * the function to sort the data
+   */
+  onSort?: () => void;
```

Comment:
@ByronDWall In the ticket it's written `onSort`, but we used the `on...Request` notation previously when buttons are actually doing nothing except triggering a callback fn.  

`onSortRequest`?

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2951/commits/87f00d071d260cc8ba05dc38380993e55887aeca

Lets discuss this too and I will follow up

Thank you

Reply:
yes, sorry, it should be `onSortRequest`

Reply:
Updated: https://github.com/commercetools/ui-kit/pull/2951/commits/ec5a8a9aed833d452099407e0cb097c85b871fcd

Thanks